tests/test_weather_plugin.py

```
@@ -1113,21 +1240,51 @@ def test_generate_forecast_requests_exception(self, mock_get):
+
+        async def run_test():
+            mock_packet = {
+                "decoded": {
+                    "portnum": TEXT_MESSAGE_APP,
+                    "text": "!weather",
+                },
+                "channel": 0,
+                "to": "abc",
+                "fromId": "node1",
+            }
+            mock_room = MagicMock()
```

Ruff (0.14.6)

Local variable `mock_room` is assigned to but never used

Remove assignment to unused variable `mock_room`

(F841)